### DISCUSSION TITLE
Killing a PID - how to know which temp files to delete?

### DISCUSSION THREAD

**Discussion by paullepa** [swar/Swar-Chia-Plot-Manager#356] [General]
Hi There:
In tweaking my plotter config (yaml) files, I inevitably find I'm filling up my HD to 100%. I'm trying to figure out how I can kill one process, then delete the temporary files associated with the process.

I'm running Windows, and the log files show the temp files they use, but how do I figure out which process ID is using which log file?

**Comment by intdsgn** [comment]
The plot temp files have a syntax that begins with the date and time.
I just match the time started in Plot Manager and remove the temp files associated with that time.

Example:
The PID you want to kill has a started time of 05/14 at 2:52.
The temp file will begin as such: plot-k32-2021-05-14-14-52
Delete those. Don't forget to empty the Bin.

Hope this helps.
